docker/dev/config_docker.json

```diff
@@ -82,6 +82,14 @@
         "ShowEmailAddress": true,
         "ShowFullName": true
     },
+    "LegalAndSupportSettings": {
```

Comment:
Same 0/5, only thoughts towards keeping it as is, is if we keep the `SupportEmail` field in there, or if we plan to put other settings there non-link related in the future.